Unix/regress

```diff
@@ -118,7 +118,29 @@ root=`dirname $0`
 ##==============================================================================
 cleanup() 
 { 
-    ./tools/auth_tests_cleanup.sh $1
+    ./tools/auth_tests_cleanup.sh $1 $2
+}
+
+##==============================================================================
+##
+## Create soft-link to $OMI_USER account
+##
+##==============================================================================
+createlink()
+{ 
+    username=$OMI_USER
+    new_output="/tmp/$USER-$current_output"
+
+    if [ "x$username" != "x" ]; then
+        userhome=`eval echo ~$username`
+        if [ -w $userhome ]; then
```

Comment:
The `-w` flag only checks DAC, not for read-only file system.  Do you want to catch an error down at 142, or is this too unlikely? If too unlikely, then why bother checking for `-w` here?